docs/integration/express.js.md

```diff
@@ -349,6 +349,272 @@ pm2 start app.js -i max
+
+// Start monitoring metrics
+stMonitor.start()
+
+
+// ...
+
+// At the top of your routes add the stHttpLoggerMiddleware to send HTTP logs to Sematext
+const express = require('express')
+const app = express()
+app.use(stHttpLoggerMiddleware)
+
+// ...
+```
+
+Then, use this `Dockerfile` to make sure garbage collection metrics are enabled:
+
+```dockerfile
+FROM alpine AS build
+WORKDIR /usr/src/app
+RUN apk add --no-cache --update \
+  python3 \
+  make \
+  gcc \
+  g++ \
+  nodejs \
+  nodejs-npm
+ 
+COPY package.json package-lock.json ./
+RUN npm install --production
+ 
+#############
+ 
+FROM alpine
+WORKDIR /usr/src/app
+RUN apk add --no-cache --update nodejs
+COPY --from=build /usr/src/app/node_modules ./node_modules
+COPY . .
+EXPOSE <PORT>
+CMD ["node", "app.js"]
+```
+
+Change the `<PORT>` you're exposing and if needed, change `app.js` to the entrypoint of your application.
+
+Now build a Docker image from the `Dockerfile` above. Run this command in the same directory where you have the `Dockerfile`:
+
+```bash
+docker build -t <YOUR_IMAGE:TAG> .
+```
+
+### Docker
+
+Next, run the Docker image. Add your `MONITORING_TOKEN`. Change the `<PORT>` to the one you're exposing and `<YOUR_IMAGE:TAG>` to the name of the image you just built. Optionally, add additional flags if you need to.
+
+```bash
+docker run -d -p <PORT>:<PORT> \
+  -v /var/run/docker.sock:/var/run/docker.sock \
+  -e MONITORING_TOKEN=<MONITORING_TOKEN> \
+  -e LOGS_TOKEN=<LOGS_TOKEN> \
+  <YOUR_IMAGE:TAG>
+```
+
+### Docker Compose
+
+Alternatively, you can use Docker Compose. Use this `docker-compose.yml` file alongside the `Dockerfile` above to build and run your application. First add your `<MONITORING_TOKEN>`, then change the `<PORT>` to the one you're exposing and `<YOUR_IMAGE:TAG>` to the name of the image you just built.
+
+```yaml
+version: '3'
+services:
+  your-nodejs-app:
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    image: '<YOUR_IMAGE:TAG>'
+    environment:
+      - MONITORING_TOKEN=<MONITORING_TOKEN>
+      - LOGS_TOKEN=<LOGS_TOKEN>
+    restart: always
+    volumes:
+      - '/var/run/docker.sock:/var/run/docker.sock'
+    ports:
+      - '<PORT>:<PORT>'
+```
+
+### Docker Swarm
+
+The same approach works for Docker Swarm. First add your `<MONITORING_TOKEN>`, then change the `<PORT>` to the one you're exposing and `<YOUR_IMAGE:TAG>` to the name of the image you just built.
+
+```bash
+docker service create --name your-nodejs-app \
+  -p <PORT>:<PORT> \
+  --restart-condition any \
+  --mount type=bind,src=/var/run/docker.sock,dst=/var/run/docker.sock \
+  -e MONITORING_TOKEN=<MONITORING_TOKEN> \
+  -e LOGS_TOKEN=<LOGS_TOKEN> \
+  <YOUR_IMAGE:TAG>
+```
+
+Alternatively, if you want to use a Docker Compose file with the `docker stack` command, add this to your `docker-compose.yml` file. First add your `<MONITORING_TOKEN>`, then change the `<PORT>` to the one you're exposing and `<YOUR_IMAGE:TAG>` to the name of the image you just built.
```

Comment:
> add this

What is this referring to?  Stuff above or below?

Reply:
Re-worded to say: `...snippet below...`